code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -124,12 +124,7 @@
 
 /obj/structure/closet/crate/necropolis/bubblegum/PopulateContents()
 	new /obj/item/clothing/suit/hooded/hostile_environment(src)
-	var/loot = rand(1,2)
```

Comment:
Dont remove it, comment it out. Use /* */ block comment and name it Bubberstation removal start at the begining and at the end